src/components/m-config-vue3.vue

```diff
@@ -20,37 +20,50 @@ export default defineComponent({
   components: {
     Vue3JsonEditor, 
   },
-  setup () {
+  props: ['session'], 
+  setup (props) {
     function onJsonChange (value) {
       console.log('value:', value)
+      state.json = value; 
+    }
+    function currentConfig(){
+      return state.json; 
+    }
+ 
+    var state; 
+    if (props.session === null){
+      state = reactive({
+        json: {'batch_size': 64,
```

Comment:
lets leave this out. of this code. which config is used as default will come from the server as well. so assume the component will get the right config at initialization.